packages/api/src/context.ts

```diff
@@ -269,8 +276,10 @@ export async function contextFromRequest(
               console.error(peer)
               return null
             }
-
-            const fetcher = createFetcher(peer.hostURL, peer.token)
+            const peerTimeout = Number(
+              (await dbAdapter.setting.getSetting(SettingName.PEERING_TIMEOUT_MS))?.value
```

Comment:
what happens, if the value is undefined or null? what would Numer() return? And how would the createFetcher react?

Reply:
I can use a fallback to the env variable, but that's not guaranteed to be defined either. Would it be better to throw an error or to set a default value?